ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -23,8 +23,16 @@ QtObject {
 
     readonly property var accounts: walletSectionAccounts.accounts
 
-    property var fromNetworksModel: walletSectionSendInst.fromNetworksModel
-    property var toNetworksModel: walletSectionSendInst.toNetworksModel
+    property var fromNetworksRouteModel: LeftJoinModel {
+        leftModel: root.walletSectionSendInst.fromNetworksRouteModel
+        rightModel: root.flatNetworksModel
+        joinRole: "chainId"
+    }
```

Comment:
Ideally store should by only thin wrapper hiding context properties exposed from the backend. Data transformations should be kept `adaptor` objects, responsible only for preparing data for one or multiple views or other adaptors.

Another thing: there is related stub defining `flatNetworksModel` - so probably it affects some storybook pages. Should be refactored appropriately as well.